pvmismatch/docs/conf.py

```diff
@@ -29,7 +29,9 @@
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx',
               'sphinx.ext.mathjax', 'sphinx.ext.napoleon',
-              'sphinx.ext.graphviz']
+              'sphinx.ext.graphviz', 'sphinx.ext.imgconverter']
+
+image_converter = 'c://Program Files//ImageMagick-7.0.7-Q16//convert.exe'
```

Comment:
where is this program running?

Reply:
[ImageMagick](https://www.imagemagick.org/script/index.php) is used by Sphinx to convert the Travis `.svg` badge into an image that can be used in a pdf and latex. I installed it, and it works. It's available on all platforms.